sympy/physics/control/lti.py

```diff
@@ -249,6 +250,38 @@ def expand(self):
         """
         return TransferFunction(expand(self.num), expand(self.den), self.var)
 
+    def dcgain(self):
+        """
+        Returns the zero-frequency (or DC) gain. It is the transfer function
+        obtained by taking the limit as ``var`` goes to zero.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(s + 3, s**2 - 9, s)
+        >>> tf1.dcgain()
+        TransferFunction(-1, 3, s)
```

Comment:
If s -> then you should get a number not a transfer function.

Reply:
Returning a number instead of a transfer function was my first choice, but I changed it for two reasons:
 1. In https://github.com/sympy/sympy/pull/19390#discussion_r439570137, you did return a transfer function instead of a number.
```
>>> closed_loop.zero_frequency_gains()  # takes limit as s -> 0
TransferFunction(...) 
```
 2. Although other libraries return a number, but this being a symbolic package, and we can get certain values like `(a + b) / c` (`a`, `b`, `c` are `symbols`) after taking a limit so I thought wrapping a transfer function around it was a better choice.

Reply:
> In #19390 (comment), you did return a transfer function instead of a number.

That was likely me not thinking carefully about it.

> Although other libraries return a number, but this being a symbolic package, and we can get certain values like (a + b) / c (a, b, c are symbols) after taking a limit so I thought wrapping a transfer function around it was a better choice.

Sure, it will be an expression if the values are symbols. If it evaluates to numbers, then it will return a Mul representing a ratio of two numbers I assume.

Either way it shouldn't return a transfer function because it is no longer a function of s.